src/components/MapExplorer.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default React.createClass({
+  render() {
+    return (
+      <div id="map-explorer">
```

Comment:
I'd prefer using classes to IDs for consistency. I think we only use `#app-container` to target the root element for react efficiently